sunkit_image/nafe.py

```diff
@@ -0,0 +1,411 @@
+"""
+Copyright (c) 2019 Mina Sami
+This file is part of SunPy/sunkit-image.
+Check LICENSE fore more details.
+
+This module is an implementation of:
+@article{0067-0049-207-2-25,
+  author={M. Druckmüller},
+  title={A Noise Adaptive Fuzzy Equalization Method for Processing Solar
+         Extreme Ultraviolet Images},
+  journal={The Astrophysical Journal Supplement Series},
+  volume={207},
+  number={2},
+  pages={25},
+  url={http://stacks.iop.org/0067-0049/207/i=2/a=25},
+  year={2013},
+  abstract={A new image enhancement tool ideally suited for the visualization
+  of fine structures in extreme ultraviolet images of the corona is presented
+  in this paper. The Noise Adaptive Fuzzy Equalization method is particularly
+  suited for the exceptionally high dynamic range images from the Atmospheric
+  Imaging Assembly instrument on the Solar Dynamics Observatory. This method
+  produces artifact-free images and gives significantly better results than
+  methods based on convolution or Fourier transform which are often used for
+  that purpose.}
+}
+"""
```

Comment:
We don't need this starting string, if you want to add a reference, I would suggest adding to the main docstring of the function, as a bullet point. You should see an example in the noise.py file.

Reply:
Changed to previously mentioned example and will add link to the paper as in given example.